src/components/POLine/POLineDetails/POLineDetailsForm.js

```diff
@@ -48,6 +48,7 @@ function POLineDetailsForm({
   integrationConfigs = [],
 }) {
   const createInventorySetting = getCreateInventorySetting(get(parentResources, ['createInventory', 'records'], []));
+  const isManualOrder = isOrderManual(order);
```

Comment:
Do you really need `isOrderManual` func? `const isManualOrder = Boolean(order?.manualPo)`

Reply:
Not really)) It may be omitted, thanks